### PR TITLE
feat(digitial-passes): Tutorial on Apple Wallet digital passes

### PR DESCRIPTION
**Pull request by dtarnawsky**
This is a tutorial on creating and adding digital passes with Apple Wallet. Some details are there for Google Wallet which will be fleshed out later when I have plenty of time to create a working example with Google Wallet.

### CODE REVIEW

File: docs/integrations/digitial-passes/apple-wallet-passes/creating.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+1. **[Designing](#designing)** - Similar to HTML, a pass needs to be designed to show the information you want on it.
+2. **[Signing](#signing-a-pass)** - A pass file needs to signed to prove that it was issued by your app.
+3. **[Delivering](#test-the-pass)** - Sending the pass to your app via an API.
+
+## Designing
+Apple documents the manual [content and design of a pass](https://developer.apple.com/documentation/walletpasses/creating_the_source_for_a_pass) but there are 3rd party designers like [Passkit Visual Designer](https://pkvd.app/) to help.
+
+You can use the sample project as a design template: The [model](https://github.com/dtarnawsky/cs-digital-pass/tree/main/backend/models) folder contains the necessary images and `pass.json` file that defines the structure of the pass:
+
+```json
```
Comment (eric-horodyski) [inline docs/integrations/digitial-passes/apple-wallet-passes/creating.md:21]: You can display code file names with CodeHike. They adjust MDX in such a way that the following works without using any specific CodeHike components

```
```json models/pass.json
```

**Review by kensodemann — APPROVED** [review]

**Review by eric-horodyski — CHANGES_REQUESTED** [review]

**Comment by dtarnawsky** [comment]
> The only question I had was whether we should like to `capacitor-pass-to-wallet`, but then I went to their NPMJS entry, and there isn't really much out there of use, so meh... 🤷🏻‍♂️
> 
> LGTM

I've added a link to the repo for the plugin.

File: docs/integrations/digitial-passes/apple-wallet-passes/adding.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  public async get(url: string): Promise<string> {
+    const response = await fetch(url);
+    const blob = await response.blob();
+    const base64 = await blobToBase64(blob);
+    if (!base64 || base64 instanceof ArrayBuffer) {
+      throw new Error(`Unable to get ${url}`);
+    }
+    return base64;
+  }
+```
```
[the remark this answers is not in the document]
Comment (dtarnawsky) [reply docs/integrations/digitial-passes/apple-wallet-passes/adding.md]: Thanks, fixed it (was a copy of the function written in Angular but redid it to be more generic)

**Review by eric-horodyski — APPROVED** [review]

**Comment by marlon-ionic** [comment]
LGTM…It's not a huge deal, but I kinda think the steps should be flipped so `Creating Passes` is first. I understand that the `Adding Passes` step is simpler to follow, but If a user has to rely on this tutorial, I think they should be able to follow the steps in the order provided to complete the process.

**Review by marlon-ionic — APPROVED** [review]

**Pull request merged**